client/src/app/pages/issues/issue-detail-drawer/file-incidents-detail-modal/file-all-incidents-table.tsx

```diff
@@ -124,7 +124,7 @@ export const FileAllIncidentsTable: React.FC<
                     {...getTdProps({ columnKey: "message" })}
                   >
                     <ReactMarkdown components={markdownPFComponents}>
-                      {`${incident.message.split("\n")[0]} ...`}
+                      {`${getFirstNonEmptyLine(incident.message)} ...`}
                     </ReactMarkdown>
```

Comment:
Maybe more like "if message is blank (i.e. only has whitespace, is empty string, or is falsey), then render a "no message" placeholder else render the `ReactMarkdown` component.

Then `getFirstNonEmptyLine()` can be assured there is a line to find!